docs/MOLECULE_SCENARIOS.md

```diff
@@ -868,6 +868,8 @@ Kafka Broker Customer Listener
 
 RBAC Additional System Admin.
 
+SSO in Control center using Azure
```

Comment:
OIDC and SAML are 2 well know methods for SSO. And we are using OIDC.
Should we also highlight it here so make it more helpful for future authors / readers?

Something like : "SSO authentication using OIDC in Control center using Azure IdP"

Reply:
thanks, will do this